boards/riscv/it8xxx2_evb/it8xxx2_evb.dts

```diff
@@ -15,6 +15,7 @@
 
 	aliases {
 		i2c-0 = &i2c0;
+		peci-0 = &peci0;
```

Comment:
Why do you need this? These sorts of aliases are generally copy/pasted from older board devicetrees that were written before we had APIs like `DT_NODELABEL`. I think it should probably be removed from this PR. I'm not sure why there's an `i2c-0` alias above, either, but that is not related to this PR.

Reply:
Dear Marti, 

Thanks for response.
The alias is added for the sample code. According to the line 190 in the main.c which can be found in the directory "samples/drivers/peci/src":

        peci_dev = DEVICE_DT_GET(DT_ALIAS(peci_0));

In my personal test code, I did use the DT_NODELABEL instead of ALIAS.
Besides, the reason why there's an i2c-0, I believe that it's from another driver module which could probably encounter the same scenario.

Thanks again.

Best regards,
BJ

Reply:
Thanks for the URL, sir. And yes, it's the sample code which I mentioned and built with. So is it required to remove the alias? Does it mean that we need to replace all the DT_ALIAS with DT_NODELABEL in the future?

Reply:
Thank you -- this makes sense. I appreciate the explanation.

Reply:
Thanks again for your guidance and patience. Truly grateful. :-D